lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -173,7 +173,16 @@ bool CommandObjectDWIMPrint::DoExecute(StringRef command,
     auto *exe_scope = m_exe_ctx.GetBestExecutionContextScope();
     ValueObjectSP valobj_sp;
     ExpressionResults expr_result =
-        target.EvaluateExpression(expr, exe_scope, valobj_sp, eval_options);
+        target.EvaluateExpression(expr, exe_scope, valobj_sp, eval_options, &m_fixed_expression);
+    
+    // Only mention Fix-Its if the command applies them.
+    // The compiler errors can address any parsing issues after applying Fix-It(s).
+    if (!m_fixed_expression.empty() && target.GetEnableNotifyAboutFixIts()) {
+      Stream &error_stream = result.GetErrorStream();
+      error_stream.Printf("  Applying Fix-It to expression, changing it to:\n    %s\n",
```

Comment:
Is this printed before or after the output? If it's printed after, past tense might make more sense?

Reply:
This is the _only_ output.

Technically the print comes after the change, but it's all happening "now", in the present tense, from the developers perspective.

I can change it back to past tense, but I don't know that detail would be relevant to anyone other than someone working on LLDB itself. Even then, that person can just step through debugger and see when the change happened relative to the print statement.

Reply:
Changed phrasing to past tense because this message appears after the expression's result.

```cpp
error_stream << "  Evaluated this expression after applying Fix-It(s):\n";
error_stream << "    " << fixed_expression << "\n";
```